packages/core/src/components/fields/SchemaField.tsx

```diff
@@ -54,7 +54,7 @@ function getFieldComponent<T = any, S extends StrictRJSFSchema = RJSFSchema, F e
   const field = uiOptions.field;
   const { fields, translateString } = registry;
   if (typeof field === 'function') {
-    return field;
+    return field as React.FC;
```

Comment:
Don't use `React.FC`. And why do you need this change?

Reply:
I've reverted the changes in this file, I do not know why this was made by @MilanKrupa